docs/reference/api.md

```diff
@@ -1,15 +1,17 @@
 # API
 
-Pebble exposes an HTTP API that remote clients can use to interact with the daemon. The API has endpoints for starting and stopping services, adding configuration layers to the plan, and so on.
-
-The API uses HTTP over a Unix socket, with access to the API controlled by user ID. If `pebble run` is started with the `--http <address>` option, Pebble exposes a limited set of open-access endpoints (see {ref}`api-access-levels`) using the given TCP address.
+```{include} /reuse/api.md
```

Comment:
I don't think we should use re-use here. I think we should leave this detailed explanation in the api.md reference, and just have a one or two-sentence summary in the how-to, with a link to the API reference for more info. They're different kinds of docs, so I don't think we should re-use reference content directly in the how-to.